src/Bundle/Tests/Functional/Maker/MakeGridTest.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\GridBundle\Tests\Functional\Maker;
+
+use App\Entity\Price;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\MakerBundle\Exception\RuntimeCommandException;
+use Symfony\Component\Console\Tester\CommandTester;
+
+final class MakeGridTest extends MakerTestCase
+{
+    /** @test */
+    public function it_can_create_grids(): void
+    {
+        $tester = new CommandTester((new Application(self::bootKernel()))->find('make:grid'));
+
+        $this->assertFileDoesNotExist(self::tempFile('Grid/PriceGrid.php'));
+
+        $tester->execute(['entity' => Price::class, '--namespace' => 'Tests\Tmp\Grid']);
+
+        $this->assertFileExists(self::tempFile('Grid/PriceGrid.php'));
```

Comment:
Can we extract `'Grid/PriceGrid.php'` to const, so we will be sure that the path is identical?

Reply:
Good idea, on this class or in MarkerTestCase?

Reply:
Done on this class, cause it's grid related. If we add some other makers on this bundle, it shouldn't be related.